bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -503,6 +503,13 @@ private void checkCameraConnection() {
             }
             return;// ffmpeg snapshot stream is still alive
         }
+
+        // if this is an ONVIF cam, also use the connection state which is now
+        // also updated in the event receive handler
+        if (thing.getThingTypeUID().getId().equals(ONVIF_THING) && (onvifCamera.isConnected())) {
```

Comment:
```suggestion
        if (thing.getThingTypeUID().getId().equals(ONVIF_THING) && (snapshotUri.isEmpty())) {
```

This may work for your camera (feel free to test) and not affect other people's ONVIF cameras which do get a snapshot url. Of course the online and offline detection will not work for your camera, but other peoples should remain.